src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -93,19 +104,62 @@ const StreamGroup = createReactClass({
     SelectedGroupStore.toggleSelect(this.state.data.id);
   },
 
+  toggleShowLifetimeStats(showLifetimeStats) {
+    if (this.hoverWait) {
+      clearTimeout(this.hoverWait);
+    }
+
+    this.hoverWait = setTimeout(() => this.setState({showLifetimeStats}), 100);
```

Comment:
We have this constant we use for dropdown menus, can we use 
it here? https://github.com/getsentry/sentry/blob/fc218f38f5276e0e2298ee711d876a9b2c573028/src/sentry/static/sentry/app/constants/index.tsx#L189

Reply:
Yes